### DISCUSSION TITLE
При скачивании возникает ошибка

### DISCUSSION THREAD

**Discussion by uchihaamaya** [lirix360/ReadmangaGrabber#86] [Q&A]
При скачивании любой манги с mintmanga появляются ошибки. Авторизация выполнена, новый файл с cookie не помог. 
![Безымянный](https://github.com/lirix360/ReadmangaGrabber/assets/131579846/da516dc4-0d82-4822-9ed0-df32fde9285e). Лог 
[grabber_log.log](https://github.com/lirix360/ReadmangaGrabber/files/12328966/grabber_log.log)
Подскажите, пожалуйста, что не так?

**Comment by lirix360** [comment]
> При скачивании любой манги с mintmanga появляются ошибки. Авторизация выполнена, новый файл с cookie не помог. ![Безымянный](https://user-images.githubusercontent.com/131579846/260287457-da516dc4-0d82-4822-9ed0-df32fde9285e.png). Лог [grabber_log.log](https://github.com/lirix360/ReadmangaGrabber/files/12328966/grabber_log.log) Подскажите, пожалуйста, что не так?

Манга с mintmanga, в том числе указанная на скрине, скачивается. Судя по ошибке вы, возможно, слишком много скачивали, либо выставили слишком маленькую паузу между главами/файлами в настройках, либо их серверу ваш IP еще по какой-то причине не нравится.

**Reply by uchihaamaya** [reply]
В настройках стоит 10 сек. между главами и 10 сек между файлами. Это мало? При включении ВПН ничего не изменилось (дело не в IP). Что значит "много скачивали"? Ничего не скачивалось месяц.

**Reply by madmurdok** [reply]
Та же фигня. Ни впн, ни увеличение интервала не помогает :(
